dll/win32/mstask/stubs.c

```diff
@@ -0,0 +1,133 @@
+
+#include "precomp.h"
+
+WINE_DEFAULT_DEBUG_CHANNEL(mstask);
+
+#define NET_API_STATUS DWORD
+typedef HANDLE ATSVC_HANDLE, SASEC_HANDLE;
+typedef struct _AT_INFO *LPAT_INFO;
+typedef struct _AT_ENUM_CONTAINER *LPAT_ENUM_CONTAINER;
+
+HRESULT
+WINAPI
+ConvertAtJobsToTasks(void)
+{
+    UNIMPLEMENTED;
+    return E_NOTIMPL;
+}
+
+HRESULT
+WINAPI
+GetNetScheduleAccountInformation(
+  LPCWSTR pwszServerName,
+  DWORD ccAccount,
+  WCHAR wszAccount[])
+{
+    UNIMPLEMENTED;
+    return E_NOTIMPL;
+}
+
+NET_API_STATUS
+WINAPI
+NetrJobAdd(
+   ATSVC_HANDLE ServerName,
+   LPAT_INFO *pAtInfo,
+   LPDWORD pJobId)
+{
+    UNIMPLEMENTED;
+    return E_NOTIMPL;
+}
+
+NET_API_STATUS
+WINAPI
+NetrJobDel(
+    ATSVC_HANDLE ServerName,
+    DWORD MinJobId,
+    DWORD MaxJobId)
+{
+    UNIMPLEMENTED;
+    return E_NOTIMPL;
+}
+
+NET_API_STATUS
+WINAPI
+NetrJobEnum(
+    ATSVC_HANDLE ServerName,
+    LPAT_ENUM_CONTAINER pEnumContainer,
+    DWORD PreferedMaximumLength,
+    LPDWORD pTotalEntries,
+    LPDWORD pResumeHandle)
+{
+    UNIMPLEMENTED;
+    return E_NOTIMPL;
+}
+
+NET_API_STATUS
+WINAPI
+NetrJobGetInfo(
+    ATSVC_HANDLE ServerName,
+    DWORD JobId,
+    LPAT_INFO* ppAtInfo)
+{
+    UNIMPLEMENTED;
+    return E_NOTIMPL;
+}
+
+HRESULT
+WINAPI
+SAGetAccountInformation( 
+    SASEC_HANDLE Handle, 
+    const wchar_t* pwszJobName, 
+    DWORD ccBufferSize, 
+    wchar_t wszBuffer[])
```

Comment:
Use `LPWSTR`.

Reply:
`LPWSTR` implies that this is initialized and null-terminate on iput, which it is not (since it's an output buffer)

Reply:
See above